filament/src/details/BufferObject.cpp

```diff
@@ -45,6 +50,15 @@ BufferObject::Builder& BufferObject::Builder::bindingType(BindingType bindingTyp
     return *this;
 }
 
+BufferObject::Builder& BufferObject::Builder::name(const char* name, size_t len) noexcept {
+    if (!name) {
+        return *this;
+    }
+    size_t const length = std::min(len == 0 ? strlen(name) : len, size_t { 128u });
```

Comment:
Not sure it's safe to use zero as meaning strlen() because that's a legitimate value that a string_view could return.
So either you have an override that doesn't take a length and then you know you need to do a strlen(), or we need to always require the length to be passed.

Reply:
For now, let's just always require it to be passed